Packs/SecneurXThreatFeeds/Integrations/SecneurXThreatFeeds/SecneurXThreatFeeds.py

```diff
@@ -47,68 +102,121 @@ def parseIndicators(feedJson: Dict) -> Any:
-                        else:
-                            continue
-                        indicatorObj["rawJson"] = i
-                        indicatorObj["fields"] = {
-                            "firstseenbysource": createdValue,
-                            "indicatoridentification": indicatorIdValue,
-                            "verdict": "Malicious",
-                            "tags": tagList,
-                            "modified": modifiedValue,
-                            "reportedby": reportByValue
-                        }
-                        indicatorList.append(indicatorObj)
-
-                    except Exception as e:
-                        demisto.error(e)
+                        if SNXIocParse.SNX_MALWARE_TYPE_KEY in ioc_data.keys():
+                            tagList = ioc_data[SNXIocParse.SNX_MALWARE_TYPE_KEY]
+                        if SNXIocParse.SNX_DESCRIPTIONS_KEY in ioc_data.keys():
+                            descriptionValue = ioc_data[SNXIocParse.SNX_DESCRIPTIONS_KEY]
+                    else:
+                        continue
+
+                    indicatorObj = {}
+                    if indicatorValue is not None:
+                        indicatorObj["value"] = indicatorValue
+                    else:
+                        continue
+                    if indicatorType is not None:
+                        indicatorObj["type"] = indicatorType
+                    else:
+                        continue
+                    indicatorObj["rawJson"] = ioc_data
+                    indicatorObj["fields"] = {
+                        "indicatoridentification": indicatorIdValue,
+                        "verdict": SNXIocParse.SNX_MALICIOUS_KEY,
+                        "tags": tagList,
+                        "reportedby": reportByValue
+                    }
+                    if createdValue is not None:
+                        indicatorObj["fields"]["firstseenbysource"] = createdValue
+                    if modifiedValue is not None:
+                        indicatorObj["fields"]["modified"] = modifiedValue
+                    if confidenceValue is not None:
+                        indicatorObj["fields"]["confidence"] = confidenceValue
+                    if descriptionValue is not None:
+                        indicatorObj["fields"]["description"] = descriptionValue
+                    indicatorList.append(indicatorObj)
+
+                except Exception as e:
+                    raise DemistoException(e)
 
     except Exception as e:
-        demisto.error(e)
+        raise DemistoException(e)
     return indicatorList
 
 
+def parse_ioc_values(ioc_data):
+    patternKey = None
+    patternValue = None
+    try:
+        patternInfo = ioc_data[SNXIocParse.SNX_PATTERN_KEY]
+        patternInfo = patternInfo.replace('[', '')
+        patternData = patternInfo.replace(']', '')
+        patternKey = patternData.split("=")[0].strip()
```

Comment:
When you get these **`file:hashes.SHA256`** or **`file:hashes.SHA1`** types, it will not return any type that Demisto supports.